Makefile

```diff
@@ -1,4 +1,4 @@
 install: ${GOBIN}/makex
 
-${GOBIN}/makex: $(shell find -type f -and -name '*.go')
+${GOBIN}/makex: $(shell /usr/bin/find -type f -and -name '*.go')
```

Comment:
Add a check for OS and use `find` for non-Windows and the full path for Windows.